conductr_cli/test/test_conduct_members.py

```diff
@@ -0,0 +1,116 @@
+import json
+
+from conductr_cli.test.cli_test_case import CliTestCase
+from conductr_cli import conduct_members
+from unittest.mock import MagicMock
+
+
+class TestConductMembersCommand(CliTestCase):
+    conductr_auth = ('username', 'password')
+    server_verification_file = MagicMock(name='server_verification_file')
+
+    default_args = {
+        'dcos_mode': False,
+        'scheme': 'http',
+        'host': '127.0.0.1',
+        'port': 9005,
+        'base_path': '/',
+        'api_version': '1',
+        'verbose': False,
+        'quiet': False,
+        'long_ids': False,
+        'conductr_auth': conductr_auth,
+        'server_verification_file': server_verification_file,
+        'role': None
+    }
+
+    filtered_by_role_asdf_args = default_args.copy()
+    filtered_by_role_asdf_args.update({'role': 'asdf'})
+
+    filtered_by_role_replicator_args = default_args.copy()
+    filtered_by_role_replicator_args.update({'role': 'replicator'})
+
+    example_json = json.loads(
+        """
+            {
+                "selfNode": {
+                    "address": "akka.tcp://conductr@127.0.0.1:9004",
+                    "uid": -2007039750
+                },
+                "members": [
+                    {
+                        "node": {
+                            "address": "akka.tcp://conductr@127.0.0.1:9004",
+                            "uid": -2007039750
+                        },
+                        "status": "Up",
+                        "roles": [
+                            "replicator"
+                        ]
+                    },
+                    {
+                        "node": {
+                            "address": "akka.tcp://conductr@127.0.0.2:9004",
+                            "uid": 1902649436
+                        },
+                        "status": "Up",
+                        "roles": [
+                            "replicator"
+                        ]
+                    }
+                ],
+                "unreachable": [
+                    {
+                        "node": {
+                            "address": "akka.tcp://conductr@127.0.0.2:9004",
+                            "uid": 1902649436
+                        },
+                        "observedBy": [
+                            {
+                                "address": "akka.tcp://conductr@127.0.0.1:9004",
+                                "uid": -2007039750
+                            }
+                        ]
+                    }
+                ]
+            }
+        """
+    )
+
+    def test_calculate_rows(self):
```

Comment:
Similarly, I think the test should mock the HTTP calls to return a particular JSON structure, and then asserting the generated output.

An example of that would be:

https://github.com/typesafehub/conductr-cli/blob/master/conductr_cli/test/test_conduct_info.py#L101

I think it's important to assert the output by mocking the stdout to ensure there's coverage to flag unexpected output changes.

This is done for `conduct info`, `conduct logs`, and `conduct events` as well.